### PR TITLE
Added TypeScript support. Added TypeScript note in README.md

### LINKED ISSUE #59: Use TypeScript
We should use TypeScript for the app. Typescript only extends JavaScript, so I see no reason why not. We can slowly implement this as we add files and convert .js files if needed.

@Vpr99 what are your thoughts on this?

This issue'd be closed when we...

1. Add Typescript standard to README.md
2. Add Typescript support to the app.

@Vpr99 mentioned that he recommends adding Typescript through babel.

### PR DESCRIPTION
**Pull request by Alex-Cannon**
Close #59 

1. Installed TypeScript support via `@babel/preset-typescript`
2. Installed Jest support for TypeScript via `yarn add jest @types/jest ts-jest`. [See this link.](https://stackoverflow.com/questions/56595053/cannot-find-name-it-in-jest-typescript)
3. Added note in README.md (We are migrating to TypeScript, so future .js, .jsx files should be .ts, .tsx)

## Test this

1. Add or modify a file to have a .ts or .tsx extension.
2. Use TypeScript in those files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ Follow these instructions to get the app up and running locally.
 * [Yarn](https://yarnpkg.com/en/docs/install)
 
 ## Contributing
-* Important note: Any merges to master are automatically deployed. You've been warned!
+* Important note: **Any merges to master are automatically deployed.** You've been warned!
```
Comment (Vpr99) [inline README.md]: Good note!

**Comment by Alex-Cannon** [comment]
@Vpr99 sure I'll do it real quick

**Comment by Alex-Cannon** [comment]
@Vpr99 alright it's done.

I had to install `@types/jest` and `ts-jest` in order for tests to work.

I converted `utils.js` and `utils.test.js` to TypeScript. The tests pass. I also ran `yarn develop` and verified that I could access a category and non-profit. I could also search as well. **So the app appeared to be functioning correctly.**

**Comment by Alex-Cannon** [comment]
Sounds great. I was thinking we'd just convert files as we work on them rather than worry about it too much. I'll merge!

**Pull request merged**